enhancements/oc/mirroring-manifests.md

```diff
@@ -105,78 +98,104 @@ mirror
 5 directories, 514 files
 ```
 
-that can be applied to the local registry with the given `oc image mirror` command.
+This filesystem can then be applied to the local registry with the given `oc image mirror` command.
 
-With this enhancement, the output format would gain an additional directory `manifests` as a sibling to the current `v2` containing manifest files that can be applied to the local cluster with:
+With this enhancement, the output format would gain an additional directory `manifests` under the base directory, a sibling to the current `v2`, containing manifest files that can be applied to the local cluster with:
 
 ```console
 $ oc apply -Rf mirror/manifests
 ```
 
 The `manifests` directory may or may not contain additional subdirectories, so the recursive `-R` should be given for future-proofing.
 
-### Pushing to disk while pushing containers to the mirror registry
```

Comment:
I think we want to retain this header to cover this use-case.

Reply:
Although maybe as `### Pushing manifests to disk while pushing containers to the mirror registry` or some such.

Reply:
This was part of my effort to get three sections per @smarterclayton comment. So I have the two sections "Pushing manifest to disk..." and "Applying manifests directly to the target cluster" but should still cover all use-cases. I think it flows better too, you push manifests to disk whether or not you're pushing containers to a mirror registry. The key point about the latter though is the need for the new --manifests-to-dir which is explained in that context.